GoogleDataTransport/GDTCORLibrary/Private/GDTCOREvent_Private.h

```diff
@@ -31,6 +34,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)writeToURL:(NSURL *)fileURL error:(NSError **)error;
 
+/** Generates incrementing event IDs based on a persistent session counter stored in user defaults.
+ *
+ * @return An event ID that is the concatenation of a session counter and event counter. For
+ * example, if this is the 104th session and the 1234th event, the returned ID will be 1041234.
```

Comment:
How is that distinguished from the 41234th event of the 10th session?

Reply:
Good question. I thought a lot about this over the weekend and I think the best solution will be to store the counter value into a file.

Reply:
I'm not following how this is resolved.

Reply:
I need to update these docs, but now that I'm just incrementing a number in a file, there's no session number, it's just a number from 0 to N, so there's no conflict.